app/src/main/java/com/example/parkingsystemkotlin/mvp/view/ConfigureParkingDialogView.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.parkingsystemkotlin.mvp.view
+
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import com.example.parkingsystemkotlin.R
+import com.example.parkingsystemkotlin.databinding.DialogParkingConfigureBinding
+import com.example.parkingsystemkotlin.listener.ConfigureParkingDialogListener
+import com.example.parkingsystemkotlin.mvp.contracts.ConfigureParkingDialogContract
+import com.example.parkingsystemkotlin.mvp.view.base.FragmentView
+import com.example.parkingsystemkotlin.utils.toast
+
+class ConfigureParkingDialogView(fragment: Fragment, private val binding: DialogParkingConfigureBinding) : FragmentView(fragment),
+    ConfigureParkingDialogContract.ConfigureParkingDialogView {
+
+    override fun closeDialog() {
+        fragment?.let {
+            val dialog: DialogFragment = it as DialogFragment
+            dialog.dismiss()
+        }
+    }
+
+    override fun getParkingSpaces(): String {
```

Comment:
you can change this by 

`    override fun getParkingSpaces(): String = binding.inputFragmentDialog.text.toString()`

to be more kotlin style :D

Reply:
Done!